examples/qml_minimal/rust/build.rs

```diff
@@ -17,6 +17,7 @@ fn main() {
             ..Default::default()
         })
         // ANCHOR_END: book_qml_module
+        .with_opts(cxx_qt_lib::build_opts())
```

Comment:
should we have a cxx-qt-build feature that defaults to doing this ? then allow developers to opt out ? As cxx_qt_lib is probably used by most apps ?

Reply:
For now this will be opt-in